src/index/updater/inscription_updater/stream.rs

```diff
@@ -18,6 +21,7 @@ lazy_static! {
 struct StreamClient {
   producer: ThreadedProducer<DefaultProducerContext>,
   topic: String,
+  transaction_cache: Cache<Txid, Option<Transaction>>,
```

Comment:
thoughts on just caching owner instead of the whole transaction?

Reply:
actually `get_inscription_by_id_unsafe` also calls `get_transaction`, so we should reuse it there too